apidoc/Titanium/Network/HTTPClient.yml

```diff
@@ -545,15 +545,15 @@ examples:
     example: |
         The following code excerpt does a simple GET request and logs the response text.
 
-             var url = "http://www.appcelerator.com";
-             var client = Ti.Network.createHTTPClient({
+             const url = "http://www.appcelerator.com";
+             const client = Ti.Network.createHTTPClient({
                  // function called when the response data is available
-                 onload : function(e) {
+                 onload : (e) => {
```

Comment:
Using an arrow func here causes babel to convert `this.responseText` to `undefined.responseText` as we're parsing the code as module and not script

Reply:
Oh snap, good catch. Actually, arrow functions don't have a `this`. So any callback functions that refer to `this.whatever` can't become arrow functions (I'm assuming mostly HTTPClient examples). If you have a reference to the client, you could call `client.responseText`, though - which I think is probably better form, though impossible to do if you're trying to refer to the client from inside a callback function passed into the constructor.

Reply:
I think its in the event as well. Should be good to track, thanks guys!